pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -55,8 +55,12 @@
                         <% questions.forEach(function(question) { %>
                         <tr>
                             <td class="align-middle">
-                                <a href="<%= urlPrefix %>/question/<%= question.id %>/">
-                                    <%= question.title %></a>
+                                <% if (question.sync_errors) { %>
+                                    <i class="fa fa-times text-danger" aria-hidden="true"></i>
```

Comment:
Should we add a tooltip or popover to these icons? Maybe a popover that says "This question is not synced. Click here to see the errors."?